################################################################################
SHORT DESCRIPTION: 
################################################################################
Remove depends on XEN_DOM0 for XEN_BACKEND.

################################################################################
LONG DESCRIPTION: 
################################################################################
Service VM are privildged guest handling device models and pv-backend instead
of dom0, so they might have backend drivers without being dom0.

NDVM and NILFVM are good examples of that.

################################################################################
CHANGELOG 
################################################################################
Original Author: unknown
Port to 3.18: Eric Chanudet <chanudete@ainfosec.com>

################################################################################
REMOVAL 
################################################################################
Removal will disable NDVM and NILFVM.

################################################################################
UPSTREAM PLAN
################################################################################
None.

################################################################################
INTERNAL DEPENDENCIES 
################################################################################
NDVM and NILFVM kernels have XEN_BACKEND in their defconfig.

################################################################################
PATCHES 
################################################################################
Index: linux-4.9.6/drivers/xen/Kconfig
===================================================================
--- linux-4.9.6.orig/drivers/xen/Kconfig
+++ linux-4.9.6/drivers/xen/Kconfig
@@ -101,7 +101,6 @@ config XEN_DEV_EVTCHN
 
 config XEN_BACKEND
 	bool "Backend driver support"
-	depends on XEN_DOM0
 	default y
 	help
 	  Support for backend device drivers that provide I/O services
